lib/APIPlugin.js

```diff
@@ -37,17 +37,11 @@ class APIPlugin {
 
 			params.normalModuleFactory.plugin("parser", parser => {
 				Object.keys(REPLACEMENTS).forEach(key => {
-					parser.plugin(`expression ${key}`, expr => {
-						const dep = new ConstDependency(REPLACEMENTS[key], expr.range);
-						dep.loc = expr.loc;
-						parser.state.current.addDependency(dep);
-						return true;
-					});
-
+					parser.plugin(`expression ${key}`, ParserHelpers.toConstantDependency(REPLACEMENTS[key]));
 					parser.plugin(`evaluate typeof ${key}`, ParserHelpers.evaluateToString(REPLACEMENT_TYPES[key]));
 				});
 				IGNORES.forEach(key => {
-					parser.plugin(key, () => true);
+					parser.plugin(key, ParserHelpers.returnTrue);
```

Comment:
it should, but probably so small if even measurable, that it is kind of premature if it doesnt have any other advantages.
except those functions get called a lot, because they would need to be created on each call, not sure if that counts for this code?